Services/UserService.cs

```diff
@@ -109,5 +109,20 @@ public static Task<UpdateResult> ChangeUserStatus(string userId, bool isActive)
 
             return found;
         }
+
+        public static async Task SendChangePasswordEmail(string email, string userId)
+        {
+            await EmailService.SendMailToAddress(email, "שינוי סיסמא", $"במידה ואתה רוצה לשלות סיסממא תמצוץ לי את הביצה {userId}");
+        }
+
+        public static async Task<UpdateResult> ChangePassword(string userId, string newPassword)
+        {
+           return await UpdateUserByFields("_id", ObjectId.Parse(userId), "password", newPassword.Hash());
+        }
+
+        public static async Task<string> getUserIdByGovId(string govId)
+        {
+            return (await repo.GetUsersByField("govId", govId))[0].id;
```

Comment:
I'd add some null checks
Something like
```suggestion
            var usersWithGovId = await repo.GetUsersByField("govId", govId);
            return usersWithGovId.Count == 1 ? usersWithGovId[0].id : null;
```